tests/common/configuration/mod.rs

```diff
@@ -34,3 +36,14 @@ pub fn get_jcli_app() -> PathBuf {
     );
     path
 }
+
+/// Gets working directory
+fn get_working_directory() -> PathBuf {
+    let mut output_directory: PathBuf = std::env::current_exe().unwrap().into();
+
+    /// current exe directory is ./target/{profile}/deps/{app_name}.exe
+    /// We would like to navigate to ./target/{profile}
```

Comment:
I wonder if cargo passes down some environment variables that could help locate the target dir.

Reply:
After reading below article:

https://doc.rust-lang.org/cargo/reference/environment-variables.html

and printing all env variables available in test. so far i couldn't find a better way